apiservers/engine/backends/container.go

```diff
@@ -33,7 +33,8 @@ import (
 	"github.com/docker/engine-api/types/container"
 	"github.com/docker/engine-api/types/strslice"
 
-	"github.com/vmware/vic/apiservers/portlayer/client/exec"
+	"github.com/vmware/vic/apiservers/portlayer/client/containers"
```

Comment:
Why change client/exec -> client/containers when also creating portlayer/exec/.. and the exec.NewContainer function?
Deliberate so that the packages/functions don't reflect each others naming on either side of the api boundary?